src/object_store/src/object/opendal_engine/gcs.rs

```diff
@@ -18,6 +18,8 @@ use opendal::Operator;
 
 use super::{EngineType, OpendalObjectStore};
 use crate::object::ObjectResult;
+
+const GCS_BUFFER_SIZE: usize = 16 * 1024 * 1024;
```

Comment:
Can you add some documentation that this is the part size used in streaming upload?